castai/resource_rebalancing_schedule.go

```diff
@@ -0,0 +1,340 @@
+package castai
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"math"
+	"time"
+
+	"github.com/castai/terraform-provider-castai/castai/sdk"
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/samber/lo"
+)
+
+func resourceRebalancingSchedule() *schema.Resource {
```

Comment:
Can you add examples of how to use new resources? Similar to https://github.com/castai/terraform-provider-castai/blob/master/examples/resources/node_configuration/resource.tf
So it will be shown tf doc page https://registry.terraform.io/providers/castai/castai/latest/docs/resources/node_configuration